src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -159,8 +159,7 @@ module.exports = class ArtworkCommercialView extends Backbone.View
     e.preventDefault()
     openMultiPageModal 'collector-faqs'
 
-  trackOpenCollectorModal: (e) ->
-    e.preventDefault()
```

Comment:
i guess this was the line that prevented it?

Reply:
exactly @mzikherman. This would be fine for modal because it was triggered with js but was killing the `a` href action.